core/src/latest_unprocessed_votes.rs

```diff
@@ -285,7 +290,10 @@ impl LatestUnprocessedVotes {
                 }
                 false
             })
-            .count()
+            .count();
+
+        slot_metrics_tracker.accumulate_transaction_errors(&sanitization_errors);
```

Comment:
Same here, won't it be confusing to accumulate these errors into slot metrics? It'd be nice if they were separated so that it was clear that they came from forwards.